docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats.rst

```diff
@@ -545,15 +545,15 @@ Here are code examples of how to use these methods with different model formats:
 
 
 **MXNet, Caffe, and Kaldi** are legacy formats that need to be converted explicitly to OpenVINO IR or ONNX before running inference.
-As OpenVINO is currently proceeding **to deprecate these formats** and **remove their support entirely in the future**,
+OpenVINO **deprecated these formats** and **removed their support entirely**,
```

Comment:
```suggestion
As OpenVINO support for these formats has been discontinued,
```
The whole fragment should be changed to state clearly that these formats are not directly supported but can be used to obtain an ONNX model, or something like that

Reply:
Change applied in https://github.com/openvinotoolkit/openvino/pull/22509/commits/ef18601b07d035c40555cb80978cca7d3fb76669